lib/src/particles.dart

```diff
@@ -304,46 +263,47 @@ class Particles {
       /* move the particle */
       if (settings['particles']['move']['enable']) {
         double ms = settings['particles']['move']['speed'] / 2;
-        p.x += p.vx * ms;
-        p.y += p.vy * ms;
+        p.x! + p.vx! * ms;
+        p.y! + p.vy! * ms;
```

Comment:
This is preventing particles from moving
```suggestion
        p.x = p.x! + p.vx! * ms;
        p.y = p.y! + p.vy! * ms;
```